templates/job_request/detail.html

```diff
@@ -301,11 +301,11 @@ <h1 class="text-3xl tracking-tight break-words font-bold text-slate-900 sm:text-
       {% #card title="Code comparison" subtitle="Compare the code used in this Job Request" %}
         {% #list_group small=True %}
           {% if code_compare_url %}
-            {% #list_group_item href=code_compare_url %}
+            {% #list_group_item href=code_compare_url class="plausible-event-name=Compare+Previous+JobRequest" %}
               Compare to the previous Job Request
             {% /list_group_item %}
             {% if code_compare_succeeded_url %}
-              {% #list_group_item href=code_compare_succeeded_url %}
+              {% #list_group_item href=code_compare_succeeded_url class="plausible-event-name=Compare+Successful+JobRequest" %}
```

Comment:
To match the event name for `Job` (sorry I missed this on my previous review!)

```suggestion
              {% #list_group_item href=code_compare_succeeded_url class="plausible-event-name=Compare+Previous+Successful+JobRequest" %}
```